### ISSUE TITLE
unselect doesn't work when pointer is empty

### ISSUE THREAD

**Issue by tborychowski** [luizstacio/select-shell#11]
Hi,
Thanks for the nice lib.
1 issue I have is: when I set pointer to an empty string - the options cannot be unchecked.
I used the code from the example in readme (with minor tweaks):

```js
const select = require('select-shell');
const list = select({
	pointer: ' ',
	pointerColor: 'yellow',
	checked: ' ☑  ',
	unchecked: ' ☐  ',
	uncheckedColor: 'grey',
	checkedColor: 'green',
	msgCancel: 'No selected options!',
	msgCancelColor: 'orange',
	multiSelect: true,
	inverse: true,
	prepend: true,
	disableInput: true
});
```

Thanks!

**Comment by luizstacio** [comment]
Sorry about the gap, I'm very busy here. please if you could try an PR I will love the help =)

I will try to test it as soon as possible!

**Comment by tborychowski** [comment]
my PR is ready, have a look when you get the chance :-)

**Comment by tborychowski** [comment]
or not

**Comment by luizstacio** [comment]
@tborychowski sorry about that, just merged your pull request #12
